client_util/Cargo.toml

```
@@ -10,7 +10,7 @@ license.workspace = true
 http = "0.2.9"
 reqwest = { version = "0.11", default-features = false, features = ["stream", "rustls-tls"] }
 thiserror = "1.0.39"
-tonic = { version = "0.8" }
+tonic = { version = "0.8", features = ["tls", "tls-roots"] }
```

This uses the native certs which seems fine for most developers. If you want you can also use the Mozilla certs (at least that's what I did for the upstream Arrow CLI):

```suggestion
tonic = { version = "0.8", features = ["tls", "tonic/tls-webpki-roots"] }
```

---

Yeah that's safer indeed